packages/node/src/policy-converter.js

```diff
@@ -286,39 +256,54 @@ export const toEndoPolicySync = (policy, policyOverride) => {
  * @privateRemarks
  * The overloads aren't strictly necessary, but they're a little nicer to
  * understand for consumers.
- * @param {LavaMoatPolicy | string | URL} policyOrPolicyPath LavaMoat policy to
- *   convert (or path to policy file)
+ * @param {LavaMoatPolicy | string | URL} [policyOrPolicyPath] LavaMoat policy
+ *   to convert (or path to policy file)
  * @param {ToEndoPolicyOptions} [options] Options for conversion
  * @returns {Promise<LavaMoatEndoPolicy>}
  * @public
  */
 export const toEndoPolicy = async (policyOrPolicyPath, options = {}) => {
   await Promise.resolve()
 
-  if (!policyOrPolicyPath) {
-    throw new TypeError('Expected a policy or policy path')
-  }
+  const policy = await loadPolicies(policyOrPolicyPath, options)
 
-  /** @type {LavaMoatPolicy} */
-  let policy
-  if (isString(policyOrPolicyPath) || policyOrPolicyPath instanceof URL) {
-    const policyPath = policyOrPolicyPath
-    policy = await readPolicy(policyPath)
-  } else {
-    const allegedPolicy = policyOrPolicyPath
-    assertPolicy(allegedPolicy)
-    policy = allegedPolicy
-  }
+  const lavamoatResources = /** @type {Resources} */ (
+    policy.resources ?? create(null)
+  )
+
+  const rootPolicyRef = policy.root?.usePolicy
+
+  /**
+   * Actual conversion starts here.
+   *
+   * This converts {@link LavaMoatPolicy.resources LavaMoat's resources property}
+   * to {@link LavaMoatEndoPolicy.resources Endo's resources property}.
+   *
+   * @type {LavaMoatEndoPolicy['resources']}
+   */
+  const resources = fromEntries(
+    entries(lavamoatResources).reduce((acc, [resourceName, resourcePolicy]) => {
+      const pkgPolicy = convertEndoPackagePolicy(
```

Comment:
(could) 
```suggestion
      const pkgPolicy = convertToEndoPackagePolicy(
```
just an itch

Reply:
- [ ] see if other functions need similar renaming